packages/base/src/util/ColorConversion.js

```diff
@@ -319,6 +319,28 @@ const HEXToRGB = hex => {
 	return rgbValues;
 };
 
+const RGBtoHEX = color => {
```

Comment:
color - either document it or destructure it like in HSLtoRGB.

Reply:
Documented it with the expected format